client/app/components/groupInfo/groupInfo.html

```diff
@@ -2,7 +2,7 @@
   <md-toolbar class="md-hue-1">
     <div class="md-toolbar-tools">
       <h1 class="md-title" flex>
-        <md-button ui-sref="groupList" class="md-icon-button">
+        <md-button ui-sref="joinGroupList" class="md-icon-button">
```

Comment:
aaaah nice. Yes sure let me give it a try. I will create the end to end test for the fix ( making sure that clicking on the button you fixed directs the user to the group list).